tools/container_run_ci

```diff
@@ -71,7 +71,10 @@ fi
 cd /opt
 ./tools/install-new-deps.sh
 export CI=1 WITH_COVER_OPTIONS=1 CHECK_GIT_STATUS=1 PERL_TEST_WARNINGS_ONLY_REPORT_WARNINGS=$PERL_TEST_WARNINGS_ONLY_REPORT_WARNINGS
-cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -S . -B \"$build_dir\"
-cmake --build \"$build_dir\" --verbose --target check-pkg-build
-cmake --build \"$build_dir\" --verbose --target $target
-cover -report html_minimal \"$build_dir\"/cover_db"
+make
```

Comment:
I couldn't get this running with cmake, as this already runs all tests with `prove`, but for this we need to run the shell script instead.
So we need a way to run cmake but without running tests, just building